corehq/motech/openmrs/repeaters.py

```diff
@@ -33,8 +35,15 @@ class Meta(object):
     friendly_name = _("Forward to OpenMRS")
     payload_generator_classes = (FormRepeaterJsonPayloadGenerator,)
 
+    location_id = StringProperty(default='')
     openmrs_config = SchemaProperty(OpenmrsConfig)
 
+    def __eq__(self, other):
+        return (
+            isinstance(other, self.__class__) and
+            self.get_id == other.get_id
+        )
```

Comment:
What's this for?

Reply:
I ask, because equality has different meanings in different contexts, like, "is this changed repeater the same as the original" is different than "do these refer to the same repeaters".